MekHQ/src/mekhq/campaign/rating/FieldManualMercRevDragoonsRating.java

```diff
@@ -314,10 +315,13 @@ private void updateDropJumpShipSupportNeeds(Entity en) {
     //   3 + (4/5) = 3 + 0.8 = 3.8 = 4 hours.
     //   total = 16 hours.
     private void calcMedicalSupportHoursNeeded() {
-        int numSquads = new BigDecimal(getCampaign().getPersonnel().size())
+        ArrayList<Person> activePersonnel = new ArrayList<>();
+        // Only count active personnel. Dead or absent people don't need medical support. >.>
+        for (Person p : getCampaign().getPersonnel()) { if (p.isActive()) { activePersonnel.add(p); } }

```

Comment:
I think we're doing this "isActive" check often enough that Campaign probably warrants a "getActivePersonnel()" method that carries the work out internally.